Core/Systems/SkillSystem/SkillPlayer.cs

```diff
@@ -1,7 +1,56 @@
-﻿namespace FunnyExperience.Core.Systems.SkillSystem
+﻿using FunnyExperience.Content.Items.Gear;
+using FunnyExperience.Content.Skills.Melee;
+using Microsoft.Xna.Framework.Input;
+using Terraria.GameInput;
+
+namespace FunnyExperience.Core.Systems.SkillSystem
 {
 	internal class SkillPlayer : ModPlayer
 	{
+		private static ModKeybind _skill1Keybind;

```

Comment:
This dosent allign well with the UI, which supports 3 hotley-based skills, tied to 3/4/5. QWERT provides numerous problems as it overlaps with vanilla inputs, W for upwards movement, E for grapple, R for mount

Reply:
I updated to three skills for now with the defaults behind toprow 3,4,5.